script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_2_2.txt

```diff
@@ -1,7 +1,7 @@
 <Page1>
 ...I drop my gaze to the floor as if to turn away from the ominous premonition. //Avoiding repetition
 <Page2>
-...This is fine.
+...This is fine
```

Comment:
Ignore this I don't have a clue on how to remove this test change from my pull requests.

Reply:
Just go back into the file and add the period back. You can then recommit with the change.
:arcsmile: